python/perfetto/trace_processor/api.py

```diff
@@ -167,16 +167,17 @@ def metric(self, metrics: List[str]):
     return metrics
 
   def trace_summary(self,
-                    specs: List[str],
+                    specs: Union[List[str], List[bytes]],
```

Comment:
AFAIK the trace processor C++ api supports *both* textproto and proto. Is there any reason we cannot do the same by doing `List[Union[str, bytes]]`